scripts/README.md

```diff
@@ -0,0 +1,13 @@
+`go-dep-mk` is a generic make file used to build, format and test Golang components.
+GNU Makefile rule almost in every case contains 2 versions:
+* local -  this rule is not launched inside docker
+* not-local - this rule is launched inside docker with component directory mounted as volume 
+Possible checks for GNU Makefile:
+* check-fmt
+* check_imports
```

Comment:
shouldn't be `check-imports`?
In addition to that, this README file does not help me to understand the current build process

Reply:
Aaaa README is under construction